boards/esp32-c3-devkitM-1/src/main.rs

```diff
@@ -149,6 +151,7 @@ unsafe fn setup() -> (
     peripherals.timg0.disable_wdt();
     peripherals.rtc_cntl.disable_wdt();
     peripherals.rtc_cntl.disable_super_wdt();
+    peripherals.rtc_cntl.enable_fosc();
```

Comment:
This is probably okay because (1) the platform is a dev kit, and (2) this board is still early in bringup, but long-term we will only want to enable this when it's actually needed to be powered on.